pkgs/os-specific/linux/cannelloni/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, cmake, lksctp-tools, sctpSupport ? true }:
+stdenv.mkDerivation rec {
+  pname = "cannelloni";
+  version = "1.1.0";
+  src = fetchFromGitHub {
+    owner = "mguentner";
+    repo = pname;
```

Comment:
```suggestion
    repo = "cannelloni";
```

Reply:
With no strong feelings either way and my "nix noob" hat on, is there a reason this is better?

Reply:
Oh wait, this is to avoid using `rec` right? Ok, I'm with you

Reply:
To avoid rec, we use the finalAttrs pattern, see my feedback further down.